DevAcademy/Scenes/PlacesLive/PlacesScene/PlacesSceneView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  PlacesSceneView.swift
+//  DevAcademy
+//
+//  Created by Tomáš Melichar on 25.07.2023.
+//
+
+import SwiftUI
+import ActivityIndicatorView
+
+struct PlacesSceneView: View {
+    @EnvironmentObject private var coordinator: Coordinator
+    
+    let viewState: PlacesSceneViewState = .init()
+    
+    var body: some View {
+        NavigationStack {
+            Group {
+                if viewState.isPlaceEmpty {
+                    List(viewState.places, id: \.properties.ogcFid) { place in
+                        NavigationLink(destination: PlaceDetailView(viewState: .init(place: place))) {
```

Comment:
tady si nejspis jenom zapomel pouzit ten coodrinator, mas uz na to nachystanou funkci `makeDetailView`. 😉

Reply:
Jejda, pravda. Opraveno :octocat: